src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -60,9 +60,7 @@ private static VoltageLevelMapData toMapData(VoltageLevel voltageLevel, boolean
                     .nominalVoltage(voltageLevel.getNominalV())
                     .lowVoltageLimit(Double.isNaN(voltageLevel.getLowVoltageLimit()) ? null : voltageLevel.getLowVoltageLimit())
                     .highVoltageLimit(Double.isNaN(voltageLevel.getHighVoltageLimit()) ? null : voltageLevel.getHighVoltageLimit());
-            if (voltageLevel.getTopologyKind().equals(TopologyKind.NODE_BREAKER)) {
-                mapVoltageLevelSwitchKindsAndSectionCount(builder, voltageLevel);
-            }
+            mapVoltageLevelSwitchKindsAndSectionCount(builder, voltageLevel);
```

Comment:
For a voltage level in bus_breaker topology, busbarCount, sectionCount and switchKinds should be null instead of being initialized to 1 and empty list ...
Doing this will minimize the json size returned to the front ...
But some changes should be done in the front, because these fields can be undefined ...